javascript/ql/lib/Expressions/ExprHasNoEffect.qll

```diff
@@ -38,6 +38,8 @@ predicate inVoidContext(Expr e) {
   )
   or
   exists(LogicalBinaryExpr logical | e = logical.getRightOperand() and inVoidContext(logical))
+  or
+  exists(ConditionalExpr cond | e = cond.getABranch())
```

Comment:
Don't you need to add `and inVoidContext(cond)`?

Reply:
🙈

Reply:
That would indeed have had a bit of an effect on results! Now fixed.